### ISSUE TITLE
Make the storage scanner respect sides

### ISSUE THREAD

**Issue by josephcsible** [McJty/RFTools#1537]
Currently, the storage scanner always interacts with the "null" (internal) side of blocks. You should instead be able to choose which sides it interacts with. In particular, when being used with XNet, it should respect the side chosen there.

**Comment by McJty** [comment]
With XNet yes. But I don't see how to do this in general? The problem is how in the user interface to specify the side. This is pretty complicated

**Comment by josephcsible** [comment]
I guess for now just make it an XNet-exclusive feature. Then once the backend is there and it works, we can revisit how to do it without it.

**Comment by josephcsible** [comment]
Also, it should be possible to see the same block multiple times, such as if you connect both the top and side of a furnace.

**Comment by LezChap** [comment]
Yes, having an issue that I think is related (Xnet and Storage Scanner)...inputting items into the Scanner via Xnet doesn't seem to respect the item filters in Xnet Storage channels, and also seems to input items into TE machine's output slots.
